src/main/java/com/mojang/brigadier/CommandDispatcher.java

```diff
@@ -231,7 +231,6 @@ public int execute(final ParseResults<S> parse) throws CommandSyntaxException {
                 if (child != null) {
                     forked |= context.isForked();
                     if (child.hasNodes()) {
-                        foundCommand = true;
```

Comment:
Looks like this was here to make sure cases where redirect modifiers returned zero results don't give any error (e.g. with `/execute as` where the selector returns no results). `foundCommand` should probably still be set to true if `modifier.apply()` returns 0 results 🙂